core/src/banking_stage/consumer.rs

```diff
@@ -458,6 +458,84 @@ impl Consumer {
         }
     }
 
+    pub fn process_and_record_aged_transactions(
+        &self,
+        bank: &Arc<Bank>,
+        txs: &[SanitizedTransaction],
+        max_slot_ages: &mut [Slot],
+    ) -> ProcessTransactionBatchOutput {
+        let (
+            (transaction_costs, transactions_qos_results, cost_model_throttled_transactions_count),
+            cost_model_us,
+        ) = measure_us!(self
+            .qos_service
+            .select_and_accumulate_transaction_costs(bank, txs));
+
+        // Check ages against slot of the bank.
+        // Check that look-up tables have not been closed.
+        // TODO: This should really be done before QoS - but requires refactor of QoS
```

Comment:
(i wonder why `ci/nits.sh` didn't yell... lol)

Reply:
🤷 this was fixed anyway with the rebase on #31542